src/types/utils.d.ts

```diff
@@ -1,15 +1,14 @@
 export interface ImageRef {
   /** The src of the image */
   src: string;
-  /*
-   * You can specify the dimensions of the image
-   *
-   * The dimensions will be taken into account and are used as **max-width** and **max-height**
-   **/
-  dimensions?: {
-    width: number;
-    height: number;
-  };
+  resolutions: Record<
```

Comment:
We may reuse the comment from src/types/sections.d.ts::FullWidthImageSliderSectionSlide["image"] for "resolutions":
```
   /**
   * A resolution map which provides the width / height and url for each resolution
   *
   * The Section will automatically make sure, that the correct image is referenced
   */
```